api/server/router/swarm/cluster_routes.go

```diff
@@ -107,7 +107,12 @@ func (sr *swarmRouter) createService(ctx context.Context, w http.ResponseWriter,
 		return err
 	}
 
-	id, err := sr.backend.CreateService(service)
+	encodedAuth := ""
+	if auth, ok := r.Header["X-Registry-Auth"]; ok {
```

Comment:
It seems like the rest of the code is also case sensitive, so the issue is orthogonal to this PR.